docs/data/joy/components/typography/typography.md

```diff
@@ -158,9 +155,9 @@ declare module '@mui/joy/styles' {
 
 :::
 
-## Remove the built-in scale
+### Removing the scale
 
-To start fresh with your own typographic scale, assign an `undefined` value to the built-in typography tokens in the theme.
+To create a custom typographic scale, you can clear the built-in values by assigning `undefined` to them in the theme.
```

Comment:
This line appears to be saying the same thing as line 113, yet I believe these sections are explaining two different concepts. Can you clarify the difference between the two?